TokenManagementSystem/public/css/student/submissions.css

```diff
@@ -1,6 +1,46 @@
+
```

Comment:
No need to put the common css in all files since all blade.php files are rendering main.css file.
So remove this common css in other files except main.css